spec/dummy/spec/system/import_file_status_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+require 'rails_helper'
+include Warden::Test::Helpers
+
```

Comment:
We seem to have a lot of system specs and relatively fewer lower level test.  It seems like the system specs are starting to duplicate one another a bit (e.g. how many system specs launch an import so far?) and expensive to run.  Over the long term, we might want to think about how we accomplish more testing at unit level or at least in ways that require less expensive setup.  see https://martinfowler.com/bliki/TestPyramid.html